src/components/CodeSearchResults/CodeSearchResultsTable.jsx

```diff
@@ -51,17 +52,37 @@ const transformSearchResults = results => results.map(({
   course_key: courseKey,
   course_title: courseTitle,
   redeemed_date: redemptionDate,
+  is_assigned: isAssigned,
+  user_email: assignedTo,
   ...rest
 }) => ({
   couponId,
   couponName,
   courseKey,
   courseTitle,
+  assignedTo,
   redemptionDate: getFormattedDate(redemptionDate),
   isRedeemed: !!redemptionDate,
+  isAssigned: !!isAssigned,
   ...rest,
 }));
 
+const handleTableColumns = (searchQuery) => {
+  const getColumnIndexForKey = key => tableColumns.findIndex(column => column.key === key);
```

Comment:
It looks like we're only trying to find the column index of `assignedTo` here. This is a bit of nitpick but, maybe instead of a function where you can specify a `key`, we just get the index of the `assignedTo` column directly and assign it to a variable, e.g.:

```
const assignedToColumnIndex = tableColumns.findIndex(column => column.key === 'assignedTo');
```

That way, we won't need to find the column index independently in each conditional below.

Reply:
Agreed. Thanks for approving the PR! I'll incorporate this new suggestion as well.